app/models/google_event.rb

```diff
@@ -1,4 +1,16 @@
 class GoogleEvent
+  class InvalidParamsException < StandardError; end
```

Comment:
I'd also rename `InvalidParamsException` to `InvalidParamsError` to follow Ruby conventions